src/react-components/auth/AuthContext.js

```diff
@@ -65,6 +65,12 @@ export function AuthContextProvider({ children, store }) {
       const socket = await connectToReticulum();
       authChannel.setSocket(socket);
       await authChannel.verifyAuthentication(authParams.topic, authParams.token, authParams.payload);
+      await fetch(getReticulumFetchUrl("/api/v1/accounts/set_cookie"), {
+        method: "GET",
```

Comment:
This is an issue on the reticulum side, but since I noticed it here. These 2 calls are mutating state, so they should not be GET requests. This isn't just a semantic issue either, it can lead to some nasty hard to debug issues when intermediate parties (rightfully) assume they can call this url as much as they like with no side effects.